CMakeLists.txt

```diff
@@ -11,7 +11,7 @@ if(MSVC)
   #       are now updated only when the object files providing the symbols are updated
   cmake_minimum_required(VERSION 3.16 FATAL_ERROR)
 else()
-  cmake_minimum_required(VERSION 3.9 FATAL_ERROR)
+  cmake_minimum_required(VERSION 3.12 FATAL_ERROR)
```

Comment:
Wasn't this supposed to be 3.16? Also, I supposed in the same PR one should search and remove hacks for older versions (like the Python setup), as that will not be necessary anymore.

Reply:
@amadio sadly we can't move directly to 3.16 because Debian 10 has only cmake 3.13 packaged https://pkgs.org/search/?q=cmake

Reply:
Ok, if you want to keep Debian 10 support, I guess 3.12 is the minimal version to be worth an update.

Reply:
I updated to 3.13 which is a bit closer to 3.16

Reply:
I'd find it reasonable to install CMake 3.16 on our Debian machines, and to expect packagers to have to do the same. After all it's not a runtime requirement.

Can people still build *against* ROOT with older CMake?

Reply:
I believe yes, but we will need to remove hacks https://github.com/root-project/root/issues/8280

Reply:
(I will test such combination with rootest.git)